lib/ansible/modules/database/postgresql/postgresql_facts.py

```diff
@@ -0,0 +1,597 @@
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+
+DOCUMENTATION = '''
+---
+module: postgresql_facts
+short_description: Gathers facts about remote PostgreSQL servers.
+description:
+   - Gathers facts about remote PostgreSQL servers.
+version_added: "2.8"
+options:
+  include_subset:
+    description:
+      - Limit collected subsets by comma separated list. Allowable values are C(version),
+        C(databases), C(settings), C(tablespaces), C(languages), C(roles), C(namespaces),
+        C(replications), C(repl_slots), C(extensions). By default, collects all subsets.
+        You can use shell-style (fnmatch) wildcard to pass groups of values (see Examples).
+    default: '*'
+  exclude_subset:
+    description:
+      - Exclude subsets from the collection. Allowable values are similar as in I(include_subset).
+        You can also use shell-style (fnmatch) wildcard. Mutually exclusive with include_subset.
+  db:
+    description:
+      - Name of database to connect.
+  port:
+    description:
+      - Database port to connect.
+    default: 5432
+  login_user:
+    description:
+      - User (role) used to authenticate with PostgreSQL.
+    default: postgres
+  login_password:
+    description:
+      - Password used to authenticate with PostgreSQL.
+  login_host:
+    description:
+      - Host running PostgreSQL.
+  login_unix_socket:
+    description:
+      - Path to a Unix domain socket for local connections.
+  ssl_mode:
+    description:
+      - Determines whether or with what priority a secure SSL TCP/IP connection
+        will be negotiated with the server.
+      - See U(https://www.postgresql.org/docs/current/static/libpq-ssl.html) for
+        more information on the modes.
+      - Default of C(prefer) matches libpq default.
+    default: prefer
+    choices: ["disable", "allow", "prefer", "require", "verify-ca", "verify-full"]
+  ssl_rootcert:
+    description:
+      - Specifies the name of a file containing SSL certificate authority (CA)
+        certificate(s). If the file exists, the server's certificate will be
+        verified to be signed by one of these authorities.
+notes:
+   - The default authentication assumes that you are either logging in as or
+     sudo'ing to the postgres account on the host.
+   - This module uses psycopg2, a Python PostgreSQL database adapter. You must
+     ensure that psycopg2 is installed on the host before using this module. If
+     the remote host is the PostgreSQL server (which is the default case), then
+     PostgreSQL must also be installed on the remote host. For Ubuntu-based
+     systems, install the postgresql, libpq-dev, and python-psycopg2 packages
+     on the remote host before using this module.
+requirements: [ psycopg2 ]
+author: "Andrew Klychkov (@Andersson007)"
```

Comment:
No need for quotes, make it a list.
```suggestion
author: 
- Andrew Klychkov (@Andersson007)
```